tests/operators/test_docker_operator.py

```diff
@@ -125,7 +125,7 @@ def test_execute_unicode_logs(self, client_class_mock):
 
         client_class_mock.return_value = client_mock
 
-        originalRaiseExceptions = logging.raiseExceptions
+        originalRaiseExceptions = logging.raiseExceptions  # pylint: disable=invalid-name
```

Comment:
Same here

Reply:
Built in python functionality: https://docs.python.org/3.5/howto/logging.html#exceptions-raised-during-logging